eox_core/edxapp_wrapper/backends/users_q_v1.py

```diff
@@ -114,6 +114,7 @@ class EdnxAccountCreationForm(AccountCreationForm):
     This form it's currently only used for validation, not rendering.
     """
 
+    # pylint: disable=too-many-positional-arguments
     def __init__(  # pylint:disable=too-many-arguments
```

Comment:
Same comment as before
```suggestion
    def __init__(  # pylint: disable=too-many-positional-arguments
```